### PR TITLE
Issue 1-2: README change suggestions 

### PR DESCRIPTION
**Pull request by FraserG01**
- Includes various other changes to the points raised in issues: 

https://github.com/edgexfoundry-holding/device-bluetooth-c/issues/1
https://github.com/edgexfoundry-holding/device-bluetooth-c/issues/4

### CODE REVIEW

**Comment by FraserG01** [comment]
@lranjbar for visibility - taken the -c away in the Makefile

**Review by lranjbar — COMMENTED** [review]
The makefile changes look compatible with the new Geneva pipelines 👍

File: scripts/Dockerfile.alpine-3.9 [hunk 1]
```diff
@@ -1,28 +1,49 @@
...
+
+FROM alpine:3.9 as builder
+MAINTAINER IOTech <support@iotechsys.com>
+
+RUN apk add --update --no-cache \
+    build-base \
+    gcc \
+    cmake \
+    make \
+    linux-headers
```
Comment (tonyespy) [inline scripts/Dockerfile.alpine-3.9:29]: Just curious, why are linux-headers being installed?

**Review by tonyespy — APPROVED** [review]
Sorry, for the delay. Changes look fine to me.

That said, I wasn't able to build the latest code as for some reason I can't make the build find my SDK build.

When CMake fails it says:

`-- Could NOT find LIBCSDK (missing: LIBCSDK_LIBRARIES LIBCSDK_INCLUDE_DIR) 
`

I've tried setting `LIBCSDK_LIBRARIES` and `LIBCSDK_INCLUDE_DIR` (i.e. as cmake says they're missing), and also `CSDK_DIR` as mentioned in the README, but neither seem to work properly.

**Pull request merged**